vuepress/docs/next/docs/consume/entando-apis.md

```diff
@@ -4,56 +4,58 @@ sidebarDepth: 2
 # Accessing Entando APIs
 
 
-Entando includes the Swagger UI for API access in a quickstart environment. This document presents an overview and how to enable and access the Swagger UI for Entando APIs.
+Entando includes the Swagger UI for API access in a quickstart environment. This document presents an overview and instructions on how to enable and access the Swagger UI.
 
 ## APIs Overview
-The Entando core exposes REST APIs for every action that can be taken in the App Builder. For example, you can use APIs to create pages, page templates or add widgets to pages. The APIs can be used to support automation, testing, or integrations with external systems as well.
+The Entando App Engine uses REST APIs to enact all the functionality inside the App Builder. For example, APIs are used to add widgets to a page or create components like pages and page templates. APIs can also be used to support automation, testing, and integration with external systems.
```

Comment:
could enact -> execute? seems to be more precise and the usual language?

Reply:
I think enact is fine here, especially as APIs don't really execute things as they just move information back and forth